### ISSUE TITLE
Can not get the newest nanopolish version

### ISSUE THREAD

**Issue by Peppie-Moony** [jts/nanopolish#538]
Hi,
I tried to install the nanopolish software under my ubuntu 17 with the commands specified in the nanopolish manual. 
However in the end it always says that I have installed the 0.5.0 version of nanopolish 
(nanopolish ist schon die neueste Version (0.5.0-1build3))
Therefore I think the nanopolish index command does not work. I always get back:
nanopolish index
usage: nanopolish [command] [options]
  valid commands: 
    --help
    consensus
    eventalign
    getmodel
    help
    methyltest
    methyltrain
    scorereads
    train-poremodel-from-basecalls
    variants
  for help on given command, type nanopolish command --help
What can I do to get the proper version installed?

**Comment by jts** [comment]
You probably have an old version on your `PATH` environment variable somewhere. If you uninstall that old version it should work correctly. Alternatively you can run the version you installed from source by providing the full path to it.

**Comment by Peppie-Moony** [comment]
Thank you for your reply.
I´m very new on all the linux and github stuff. I could solve the problem. 
On our linux system a repository was deposited which always got the old version of nanopolish. 
So the thing which happened all the time was the following.
When I had compiled the new downloaded nanopolish as described in your manual with the make command and tried to to step to the index part and typed nanopolish index. My system said that nanopolish is not installed and I should try the command "sudo apt install nanopolish". When I did this the system always got the old version from the repository.  In the end I deinstalled this version as you asked me to do, tried everything again with the exception that I always have to type "./" at the beginning of every command ("./ nanopolish index" eg). Than the system could recognise that nanopolish was installed and carried out the commands.
